plugins/doc_fragments/auth.py

```diff
@@ -49,14 +34,52 @@ class ModuleDocFragment(object):
           - Vault token. Token may be specified explicitly, through the listed [env] vars, and also through the C(VAULT_TOKEN) env var.
           - If no token is supplied, explicitly or through env, then the plugin will check for a token file, as determined by I(token_path) and I(token_file).
           - The order of token loading (first found wins) is C(token param -> ansible var -> ANSIBLE_HASHI_VAULT_TOKEN -> VAULT_TOKEN -> token file).
+      token_path:
+        description: If no token is specified, will try to read the I(token_file) from this path.
+      token_file:
+        description: If no token is specified, will try to read the token from this file in I(token_path).
+        default: '.vault-token'
+      token_validate:
+        description:
+          - For token auth, will perform a C(lookup-self) operation to determine the token's validity before using it.
+          - Disable if your token doesn't have the C(lookup-self) capability.
```

Comment:
```suggestion
          - Disable if your token does not have the C(lookup-self) capability.
```

Reply:
nice, thanks, that was moved not introduced in this PR, but good a time as any to improve it!